recipe/meta.yaml

```diff
@@ -43,16 +43,16 @@ test:
     - pytest
   requires:
     - pip
-    # For full testing:
-    #- argcomplete
-    #- attrs >=19.2.0
-    #- hypothesis >=3.56
-    #- mock
-    #- nose
-    #- pygments >2.7.2
-    #- requests
-    #- setuptools
-    #- xmlschema
+    - argcomplete
```

Comment:
why have these been uncommented, but the test commands haven't been changed?

Reply:
They can be only required for [development](https://github.com/pytest-dev/pytest/commit/86945f9a1f7cae10a9ac9ccf4b41ce9ddfabe14c) not for testing

Reply:
ok but dev requirements aren't needed in the context of building a package as far as I understand? are they used by the tests at all? if not, they should be deleted. If so, the tests should be run

Reply:
They were not listed as development in version 7.4.4 (See here), but I have removed them and updated the test section.